docs/topics/whatsnew14.md

```diff
@@ -1517,7 +1517,7 @@ to help you decide which suggestions to accept and which to ignore.
 
 ![Migration inspections](migration-inspection-wn.png)
 
-Kotlin 1.4.0 is a [feature release](kotlin-evolution.md#feature-releases-and-incremental-releases) and therefore can 
+Kotlin 1.4.0 is a [feature release](kotlin-evolution.md#language-and-tooling-releases) and therefore can 
```

Comment:
For the record, not sure that we should fix the namen for the previous 1.x.x release.
I fixed the anchors just to be sure that everything works.